src/main/environments/environment.ts

```diff
@@ -1,16 +1,11 @@
-// This file can be replaced during build by using the `fileReplacements` array.
-// `ng build` replaces `environment.ts` with `environment.prod.ts`.
-// The list of file replacements can be found in `angular.json`.
-
 export const environment = {
   production: false,
```

Comment:
Firebase information is placed in the environment file then what is the purpose of [firebase.json]

Reply:
that was automatically generated. I have deleted now